app/components/job-list-item/job-list-item.component.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../styles/variables.scss';
+
+.job-list-container {
+  width: 100%;
+  border-bottom: 1px solid #E8EBEE;
+
+  .company-logo-container {
+    padding: 18px 0 18px 20px;
+
+    .company-logo {
+      border-radius: 50%;
+      height: 80px;
+      width: 80px;
+      vertical-align: middle;
+      box-shadow: inset 0px 0px 1px 0px rgba(0, 0, 0, 0.2);
```

Comment:
Maybe we could extract this to a helper `logo-box-shadow` or something *if* this box-shadow is something we're gonna use in multiple places.